src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -71,6 +72,7 @@ item.items.integrateddynamics.facade.name=Facade
 item.items.integrateddynamics.facade.info=Combine with a block in crafting table
 item.items.integrateddynamics.bucketMenrilResin.name=Menril Resin Bucket
 item.items.integrateddynamics.crystalizedMenrilChunk.name=Crystalized Menril Chunk
+item.items.integrateddynamics.variableTransformer.name=Variable Transformer
```

Comment:
Since in the config screen this shows only the transofrmer in general and not seperate in/out , and can be enabled/disabled, so i localized it